text/0010-trits/0010-trits.md

```diff
@@ -0,0 +1,121 @@
++ Feature name: trits
++ Start date: 2019-09-20
++ RFC PR: [iotaledger/bee-rfcs#10](https://github.com/iotaledger/bee-rfcs/pull/10)
++ Bee issue: [iotaledger/bee#45](https://github.com/iotaledger/bee/issues/45)
+
+# Summary
+
+This RFC proposes a `trits` interface to standardise the usage of different underlying encodings and their conversions from one to another.
+
+# Motivation
+
+A `trit` is a digit in base `3` and is usually represented by either `-1`, `0` or `1`.
```

Comment:
Should we be more precise here and write? "A balanced ternary `trit` is a digit ...."

Reply:
Can't hurt to be more precise :)